examples/content_management_examples/public/examples/msearch/msearch_table.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { TableListView, UserContentCommonSchema } from '@kbn/content-management-table-list';
+import { useContentClient } from '@kbn/content-management-plugin/public';
+import React from 'react';
+import { SavedObjectsFindOptionsReference } from '@kbn/core-saved-objects-api-browser';
+
+const LISTING_LIMIT = 1000;
+
+export const MSearchTable = () => {
+  const contentClient = useContentClient();
+
+  const findItems = async (
+    searchQuery: string,
+    refs?: {
+      references?: SavedObjectsFindOptionsReference[];
+      referencesToExclude?: SavedObjectsFindOptionsReference[];
+    }
+  ) => {
+    const { hits, pagination } = await contentClient.mSearch<UserContentCommonSchema>({
+      query: {
+        text: searchQuery,
+        limit: LISTING_LIMIT,
+        cursor: '1',
+        tags: {
+          included: refs?.references?.map((ref) => ref.id),
+          excluded: refs?.referencesToExclude?.map((ref) => ref.id),
+        },
+      },
+      contentTypes: [{ contentTypeId: 'map' }], // TODO: improve types to not require objects here?
```

Comment:
> improve types to not require objects here?

I would say yes. `contentTypes: ['map']` seems like a better API

Q: Why don't we use `TODO_CONTENT_ID` and setup `mSearch` on the server for it? The  2 examples would then go  in pair

Reply:
Otherwise we should indicate on top of the table to install the sample data to view the search result from maps.

Reply:
> Why don't we use TODO_CONTENT_ID and setup mSearch on the server for it? The 2 examples would then go in pair

Because it isn't backed by saved objects

> Otherwise we should indicate on top of the table to install the sample data to view the search result from maps.

I hope we will soon get more types there as more content onboards. 
Also the main purposes of that example is to have an integration test.

Reply:
ok I see. But we should still add some information for those who navigate to the demo and haven't installed any sample data. It is not clear that the pages only renders "map" SO type.